feed.html

```diff
@@ -36,22 +36,13 @@
                 </a>
                 <!-- <span class="label">Profile</span> -->
             </div>
-            <div class="footer-icon-container" role="group">
-                <img src="icons/leaderboard.svg#icon">
-                <!-- <span class="label">Leaderboards</span> -->
-            </div>
+            <div></div>
             <div class="footer-icon-container" role="group">
                 <a href="calendar.html">
                     <img src="icons/calendar.svg#icon">
                 </a>
                 <!-- <span class="label">Calendar</span> -->
             </div>
-            <div class="footer-icon-container" role="group">
-                <a href="post.html">
-                    <img src="icons/share.svg#icon">
-                </a>
-                <!-- <span class="label">Feed</span> -->
-            </div>
         </div>
     </footer>
```

Comment:
This might conflict with the footer changes I made earlier, but yeah we can fix those errors.